lib/rspec/core/option_parser.rb

```diff
@@ -70,7 +70,18 @@ def parser(options)
           bisect_and_exit(argument)
         end
 
-        parser.on('--[no-]fail-fast', 'Abort the run on first failure.') do |value|
+        parser.on('--[no-]fail-fast[=COUNT]', 'Abort the run after a certain number of failures (1 by default).') do |argument|
```

Comment:
@JonRowe when I added the bisect feature and wanted to support `--bisect verbose` I couldn't get that to work and had to add the `=` in the middle.  I couldn't figure out why.  Do you want to take a look at that?